.github/rawWorkflows/gh-ci-alpini-parametrized-flow.txt

```diff
@@ -28,7 +28,7 @@
         if: steps.check_alpini_files_changed.outputs.alpini == 'true'
         with:
           java-version: ${{ matrix.jdk }}
-          distribution: 'microsoft'
+          distribution: 'temurin'
```

Comment:
Microsoft is not available for jdk8?

Reply:
Yeah. Microsoft's docs suggest to use `temurin` instead: https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-8. We could split the matrix to use `microsoft` for jdk 11 and 17, and `temurin` for jdk 8 if we wish to

Reply:
Hmm... either way is fine I guess. I can give another ship it if you prefer changing it. Your call.

Reply:
I think we can keep it this way and change it later if there are problems